front/pages/api/stripe/webhook.ts

```diff
@@ -290,7 +293,18 @@ async function handler(
                 { transaction: t }
               );
             });
-
+            if (userId) {
+              const workspaceSeats = await countActiveSeatsInWorkspace(
+                workspace.sId
+              );
+              const amplitude = getBackendClient();
+              amplitude.subscriptionCreated(`user-${userId}`, {
```

Comment:
Question: Why is a subscription event on a user id key over a workspace id ? Off the top of my head I assumed we would have user based events and workspace based events.

Reply:
On Amplitude events are attached to a user.
They can also be attached to a group, and this will be added soon, but group management on amplitude is a bit complicated and relatively new, I still have to do a full path to get it to work properly.
A few things need to be done, but mostly, the custom typed SDK from amplitude does not let you specify group id with an event, making it impossible.

It's my plan to fix that by not using the custom typed SDK, but will come up in another PR (hopefully today, but not guaranteed).